packages/core/src/types/telemetry-event.ts

```diff
@@ -0,0 +1,4 @@
+export type TelemetryEvent = {
+  eventName: string;
+  payload: object;
+};
```

Comment:
This `types/` dir is kinda tech debt - I'd just put this at the top of the `service.ts` file. Because the `common.telemetry.record(...)` function will be nicely typed, callers will almost certainly be able to use the inferred type and won't need to import this.